testaddon.go

```diff
@@ -8,9 +8,18 @@ import (
 	"github.com/bitrise-io/go-utils/log"
 )
 
-// getUniqueDir returns the unique subdirectory inside the test addon export diroctory for a given artifact.
-func getUniqueDir(path string) (string, error) {
-	log.Debugf("getUniqueDir(%s)", path)
+func getExportDir(artifactPath string) string {
```

Comment:
What will happen if there are multiple 'other' artifacts with the same name?

Reply:
Very good catch. It would simply override the previously exported one.